application/gqlgen/post.go

```diff
@@ -589,3 +589,76 @@ func (r *mutationResolver) RemovePotentialProvider(ctx context.Context, postID,
 
 	return &post, nil
 }
+
+func (r *mutationResolver) MarkRequestAsDelivered(ctx context.Context, postID string) (*models.Post, error) {
+	cUser := models.CurrentUser(ctx)
+
+	var post models.Post
+	if err := post.FindByUUID(postID); err != nil {
+		return nil, domain.ReportError(ctx, err, "MarkRequestAsDelivered.FindPost")
+	}
+
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+		"post": post.UUID,
```

Comment:
These are somewhat redundant. The current user is added automatically by the rollbar code, and the post ID is in the mutation that is automatically logged. Feel free to keep them here if you like. (Same on other mutation.)

Reply:
:+1: